client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -0,0 +1,645 @@
+    setIsRenamingSheet(false)
+    if (sheetNameBeforeRename) {
+      const { index, name } = sheetNameBeforeRename
+      setValue(`resultTallySheets[${index}].name`, name)
+      setSheetNameBeforeRename(undefined)
+    }
+  }
+
+  const addSheet = () => {
+    let newSheetName = defaultResultTallySheetName(sheets.length)
+    function doesNewSheetNameConflict() {
+      return sheets.some(
+        (_, i) => getValues(`resultTallySheets[${i}].name`) === newSheetName
+      )
+    }
+
+    let incrementToAvoidConflict = 0
+    while (doesNewSheetNameConflict()) {
+      incrementToAvoidConflict += 1
+      newSheetName = defaultResultTallySheetName(
+        sheets.length + incrementToAvoidConflict
+      )
+    }
+
+    appendSheet({
+      name: newSheetName,
+      results: {},
+    })
+    setSelectedTabId(newSheetName)
+  }
+
+  const deleteSheet = (index: number) => {
+    const numSheetsAfterDeletion = sheets.length - 1
+
+    if (numSheetsAfterDeletion === 1 || index === 0) {
+      setSelectedTabId(VOTE_TOTALS_TAB_ID)
+    } else {
+      // Open the sheet before the to-be-deleted sheet
+      setSelectedTabId(
+        getValues(`resultTallySheets[${Math.max(index - 1, 0)}].name`)
+      )
+    }
+
+    // If we drop back down to 1 sheet, rename that sheet "Sheet 1" under the hood
+    if (numSheetsAfterDeletion === 1) {
+      const remainingIndex = index === 1 ? 0 : 1
+      setValue(`resultTallySheets[${remainingIndex}].name`, 'Sheet 1')
+    }
+
+    removeSheet(index)
+  }
+
+  const saveResults = (e: BaseSyntheticEvent) => {
+    cancelSheetRename()
+    setIsEditing(false)
+    handleSubmit(onValidSubmit, onInvalidSubmit)(e)
+  }
+
+  // ---------- Handlers (END) ----------
+
+  return (
+    <ListAndDetail>
+      <List
+        search={{
+          placeholder: 'Search batches...',
+          setQuery: setSearchQuery,
+        }}
+      >
+        {filteredBatches.map(batch => (
+          <ListItem
+            key={batch.id}
+            onClick={() => selectBatch(batch.id)}
+            selected={batch.id === selectedBatchId}
+          >
+            {batch.name}
+          </ListItem>
+        ))}
+      </List>
+
+      <Detail>
+        {!selectedBatch ? (
+          <p>Select a batch to enter audit results.</p>
+        ) : (
+          <>
+            <H4>{selectedBatch.name}</H4>
+            <Tabs
+              id={selectedBatch.name}
+              onChange={(newTabId: string) => {
+                cancelSheetRename()
+                setSelectedTabId(newTabId)
+              }}
+              // Defaults to false but noting explicitly for clarity. Rendering all tabs at all
+              // times is important for react-hook-form's state management
+              renderActiveTabPanelOnly={false}
+              selectedTabId={selectedTabId}
+            >
+              <Tab
+                id={VOTE_TOTALS_TAB_ID}
+                key={VOTE_TOTALS_TAB_ID}
+                panel={
```

Comment:
How is the spacing between the tabs and the results table determined? Looking at the video, the spacing looked a bit too big, but then when I ran it locally, it looked fine. I could swear the spacing is different...
<img width="718" alt="Screen Shot 2022-10-24 at 10 17 48 AM" src="https://user-images.githubusercontent.com/530106/197586984-eaeb7045-8700-4d11-9a27-9b8313ee8b91.png">